contributions/__main__.py

```diff
@@ -0,0 +1,114 @@
+#!usr/bin/python3
+from collections import namedtuple
+import os
+import requests
+import re
+from requests.auth import HTTPBasicAuth
+
+"""Download metadata for all StevensSEC projects"""
```

Comment:
Let's change all these to `#` comments, and reserve `"""` for python docstrings.